README.md

```diff
@@ -29,16 +29,17 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
 
 ## Prometheus Operator vs. kube-prometheus vs. community helm chart
 
-The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
+Prometheus Operator - The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
```

Comment:
Same here, we can maybe instead add a small header to each section so it reads better, wdyt?

Reply:
yes, small headers would be good.    It makes it abit easier to see that there are 3 options.     (I dont know how to do small headings)

Reply:
see if its better now.   is there a way to repview my changes ?